web/src/js/components/ContentView.jsx

```diff
@@ -33,10 +33,10 @@ function ContentView(props) {
         return <MetaViews.ContentTooLarge {...props} onClick={displayLarge}/>
     }
 
-    const View = ContentViews[contentView]
+    const View = ContentViews[contentView] || ContentViews['ViewServer']
```

Comment:
Ok, that makes a ton of sense (I assume you mean server, not client, in the first sentence). Let's keep it as-is!